src/types/types.ts

```diff
@@ -1318,6 +1326,8 @@ export interface Node3DProperties {
   };
 }
 
+export type NullableFilter = { isNull: boolean };
```

Comment:
I'd suggest using already existing type.  it's on the line below:`NullableProperty<T>`

Reply:
NullableProperty is for two options T and isNull, for dataSetId.isNull there's no T option. Or is it legal to use NullablePropery<void> or something like that?

Reply:
ah, my bad. then it's fine,
just fix the linter issue